integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml

```diff
@@ -32,15 +32,15 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   ALLUXIO_JAVA_OPTS: |-
-    {{if $isSingleMaster}}-Dalluxio.master.hostname={{ $fullName }}-{{ $defaultMasterName }}{{end}} -Dalluxio.master.journal.type={{ .Values.journal.type }} -Dalluxio.master.journal.folder={{ .Values.journal.folder }} {{if $isHaEmbedded}}-Dalluxio.master.embedded.journal.addresses={{range $i := until $masterCount}}{{ $fullName }}-master-{{$i}}:19200,{{end}}{{end}} {{ .Values.jvmOptions }} {{- range $key, $val := .Values.properties }} -D{{ $key }}={{ $val }} {{ end }}
```

Comment:
for single master config why did we move the property out from here to `ALLUXIO_USER_JAVA_OPTS`?

Reply:
The thing is `ALLUXIO_USER_JAVA_OPTS` always had `alluxio.master.hostname` so it's not moved from here to ALLUXIO_USER_JAVA_OPTS.

And you are right, for single master ALLUXIO_JAVA_OPTS should have `alluxio.master.hostname`. I'll add it back.